### PR TITLE
Dreamerの学習時に予測の温度パラメータを指定可能に

### PR DESCRIPTION
**Pull request by Geson-anko**
## 概要

#44 NormalMixtureが log_piを受け取る実装だと温度パラメータを動的に変更できないのでそこの仕様変更もしました。

## Submit前の確認項目

- [x] タイトルは一目でわかるようにし、説明文は PR を簡潔に説明するようにしましたか?
- [x] あなたの PR が、異なる変更を束ねたものではなく、ひとつのことだけを行うものであることを確認しましたか?
- [x] この PR で導入されたすべての変更点をリストアップしましたか?
- [x] PR を`make run`コマンドでローカルでテストしましたか?

### CODE REVIEW

File: ami/models/components/mixture_desity_network.py [hunk 1]
```diff
@@ -104,11 +112,11 @@ def __init__(
...
+        logits = torch.stack([lyr(x) for lyr in self.logits_layers], dim=-1)
 
         if self.squeeze_feature_dim:
             mu = mu.squeeze(-2)
             sigma = sigma.squeeze(-2)
-            log_pi = log_pi.squeeze(-2)
+            logits = logits.squeeze(-2)
 
-        return NormalMixture(log_pi, mu, sigma)
+        return NormalMixture(logits, mu, sigma)
```
Comment (myxyy) [inline ami/models/components/mixture_desity_network.py:122]: すみません、理論をよく理解していませんが`NormalMixture`に突っ込む値を`logits`にしても問題ないのでしょうか？

Comment (Geson-anko) [reply ami/models/components/mixture_desity_network.py:122]: `logits`はsoftmaxにかける前の全結合層の出力テンソルなので、Normal Mixtureの中で softmaxかける実装にしたので大丈夫です。Categoricalのlogitsと同じですね。

**Review by myxyy — APPROVED** [review]
Approveします
